index.js

```diff
@@ -108,7 +108,7 @@ export async function octoherd(options) {
     octokit,
     script: octoherdScript,
     userOptions,
-    octoherdReposPassedAsFlag: !!octoherdRepos,
+    octoherdReposPassedAsFlag: octoherdRepos.length,
```

Comment:
With the change made in https://github.com/octoherd/cli/pull/66/files#diff-3d406faa7d382dc8468988af3792bf78dfd057c449283aae37ad03a68d6b5142R27 `octoherdRepos` will be at least an empty array